python/client/azure/mgmt/redhatopenshift/v2021_09_01_preview/operations/_open_shift_clusters_operations.py

```diff
@@ -883,8 +879,7 @@ def begin_update(
         **kwargs  # type: Any
     ):
         # type: (...) -> LROPoller["_models.OpenShiftCluster"]
-        """Creates or updates a OpenShift cluster with the specified subscription, resource group and
-        resource name.
+        """Updates a OpenShift cluster with the specified subscription, resource group and resource name.
```

Comment:
Thanks for noticing this, I'm starting to lose track of which resources were added for this project and which pre-existed. I'll need to keep `OpenShiftCluster` the same to avoid a breaking change.